.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,22 @@
+## Description
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)